patches/minecraft/net/minecraft/block/BlockBasePressurePlate.java.patch

```diff
@@ -0,0 +1,24 @@
+--- ../src-base/minecraft/net/minecraft/block/BlockBasePressurePlate.java
++++ ../src-work/minecraft/net/minecraft/block/BlockBasePressurePlate.java
+@@ -1,6 +1,7 @@
+ package net.minecraft.block;
+ 
+ import java.util.Random;
++
+ import net.minecraft.block.material.Material;
+ import net.minecraft.block.state.IBlockState;
+ import net.minecraft.creativetab.CreativeTabs;
+@@ -120,6 +121,13 @@
+         boolean flag = p_180666_4_ > 0;
+         boolean flag1 = j > 0;
+ 
++        if (flag != flag1) 
++        {
```

Comment:
You have this check EVERYWHERE just move it to the hooks class and all your patches magically become one liners.